modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/FlaskConnexionCodegen.java

```diff
@@ -317,6 +316,15 @@ public void preprocessSwagger(Swagger swagger) {
                                     controllerPackage + "." + toApiFilename(tag)
                             );
                         }
+                        for (Parameter param: operation.getParameters()) {
+                            // sanitize the param name but don't underscore it since it's used for request mapping
```

Comment:
Flask is a little special in that the swagger file is part of the generated code so this code is to sanitize the param name in the yaml file itself.